examples/styling/raw-image.html

```diff
@@ -2,7 +2,7 @@
 <html>
```

Comment:
As we renamed other files, I would also change it to `external-image.html` :)

Reply:
Just realized we already have an `image-external` one, so I would delete this file because it is duplicated, right?

Reply:
ahhh! sorry! so i named all of the image related ones with the word `image-` first so they were easier to find and organize. 

i don't see `external-image` in my branch locally (weird)

anyway, the three files for these examples are:
<img width="167" alt="screen shot 2018-07-25 at 7 46 53 am" src="https://user-images.githubusercontent.com/1566273/43204824-273f7c82-8fdf-11e8-99b4-f8d7ee03ac5b.png">